lib/barquinhos_web/live/game_live.ex

```diff
@@ -135,8 +135,12 @@ defmodule BarquinhosWeb.GameLive do
     if "#{x}#{y}" in points, do: "ship"
   end
 
-  defp shot_class(shots, {x, y}) do
-    if {x, y} in shots, do: "shot"
+  defp shot_class(shots_received, points, {x, y}) do
+    cond do
```

Comment:
This is great! There are like 20 million ways to write conditional logic in Elixir. If you want to explore some of them, check out this doc https://elixir-lang.org/getting-started/case-cond-and-if.html and think about some ways you could re-write the same logic with different code.